it("should throw a runetimeError when trapped", () => {
  const m = WebAssembly.instantiateFromSource(watfmodule);

  assert.throws(
    m.exports.trap,
    WebAssembly.runtimeError,
    "Execution has been trapped"
  );
});
